workflow/rules/common.smk

```diff
@@ -28,9 +28,7 @@ validate(samples, schema="../schemas/samples.schema.yaml")
 
 ### Read and validate units file
 
-units = pd.read_table(config["units"], dtype=str).set_index(
-    ["sample", "unit", "run", "lane"], drop=False
-)
+units = pd.read_table(config["units"], dtype=str).set_index(["sample", "unit", "run", "lane"], drop=False)
```

Comment:
Did you run a snakefmt on it?

Reply:
Yes, it said to change it to this.
snakefmt --compact-diff -l 130

Reply:
ok, maybe I have a different version. Which version do you use?

Reply:
0.4.0

Reply:
Ah, ok, I see they have 0.4.3 as latest. But could be it doesn't change anything.